README.md

```diff
@@ -39,15 +72,27 @@ Each application should also have a `project.json` file defining an `nx` target
 
 ## Development
 
-To build the container locally: \
+To build the image locally: \
 `docker build -t aligent/nx-pipe .`
 
-To run the image locally and mount a local directory to the /app folder: \
+To run the container locally and mount current local directory to the /app/work folder:
 
-`` docker run -it -e BITBUCKET_CLONE_DIR=/app/work -e STAGE=stg -e PROFILE=bitbucket-deployer -e CFN_ROLE=test-cfn-role -v `pwd`:/app/work --memory=4g --memory-swap=4g --memory-swappiness=0 --cpus=4 --entrypoint=/bin/bash {built container id}
- ``
+```bash
+docker run -it -v $(pwd):/app/work --memory=4g --memory-swap=4g --memory-swappiness=0 --cpus=4 --entrypoint=/bin/sh \
+  -e BITBUCKET_CLONE_DIR=/app/work \
+  -e PROFILE=bitbucket-deployer \
+  -e STAGE=stg \
+  -e AWS_ACCESS_KEY_ID=test-access-key-id \
+  -e AWS_SECRET_ACCESS_KEY=test-secret-access-key \
+  -e CFN_ROLE=test-cfn-role \
+  -e UPLOAD_BADGE=true \
```

Comment:
If we're running this locally wouldn't we set the upload badge to false? Otherwise we have to have to pass a valid Bitbucket user and password

Reply:
Yeah, I set it to true to test badge generation & catch unhandled rejections :)

Reply:
@TheOrangePuff I've added the hooks. Could you please double check if it's all good?